tests/integration/src/compiler_test.rs

```diff
@@ -182,7 +182,7 @@ pub struct CompilerTestBuilder {
     /// The extra MASM modules to link to the compiled MASM program
     link_masm_modules: LinkMasmModules,
     /// Extra flags to pass to the midenc driver
-    midenc_flags: Vec<Cow<'static, str>>,
+    midenc_flags: Vec<String>,
```

Comment:
Any particular reason for this change? Most of the flags we pass are constant strings.